string id
---
float32[] rewards
int32[] results
